tests/Plugin/InvokeStrategy/OnEventStrategyTest.php

```diff
@@ -34,4 +37,25 @@ public function it_invokes_the_on_event_method_of_the_handler(): void
 
         $this->assertSame($customEvent, $onEventHandler->getLastMessage());
     }
+
+    /**
+     * @test
+     */
+    public function it_can_be_attached_to_event_bus(): void
+    {
+        $onEventStrategy = new OnEventStrategy();
+
+        $bus = $this->prophesize(EventBus::class);
+        $bus->attach(Argument::type('string'), Argument::type('callable'), Argument::type('integer'))
+            ->shouldBeCalled()
+            ->willReturn(
+                new DefaultListenerHandler(
+                    function () {
+
+                    }
```

Comment:
`{}` on the same line would be against PSR-2 (not sure whether this repository is following PSR-2).